Neos.Flow/Classes/Property/PropertyMapper.php

```diff
@@ -138,7 +138,7 @@ public function convert($source, $targetType, PropertyMappingConfigurationInterf
      */
     public function getMessages()
     {
-        return $this->messages;
+        return $this->messages ?? new Result();
```

Comment:
This could break code that checked for `$propertyMapper->getMessages() === null`

Reply:
Instead we could adjust 
```php
$this->validationResults = $this->propertyMapper->getMessages() ?? new Result();
```
in Argument::setValue()

Reply:
Done